src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.servises.AddToStorage;
+import core.basesyntax.servises.impl.AddToStorageImpl;
+import core.basesyntax.servises.impl.ReadFromFileImpl;
+import core.basesyntax.servises.impl.WriteToFileImpl;
+import core.basesyntax.storage.Storage;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        AddToStorage add = new AddToStorageImpl();
+        ReadFromFileImpl read = new ReadFromFileImpl();
+        WriteToFileImpl writeToFile = new WriteToFileImpl();
+        List<String> strings = read.readFromFile(
+                "src/main/java/core/basesyntax/resourses/input.csv");
+        add.addToStorage(strings);
+        writeToFile.addToFile(

```

Comment:
Good, but we ned DAO package 
we need use dao to work with storage , not directly